src/components/organisms/NavBar/Navbar.js

```diff
@@ -20,9 +22,9 @@ const NavBar = () => {
           <SocialButton
             to="https://www.instagram.com/makronkovo/?hl=en"
             marginRight="15px"
-            icon={['fab', 'instagram']}
+            src={InstagramIcon}
           />
-          <SocialButton to="https://www.facebook.com/makronkovo/" icon={['fab', 'facebook']} />
+          <SocialButton to="https://www.facebook.com/makronkovo/" src={FacebookIcon} />
```

Comment:
the Icon should be handled inside the component... so we will have the `icon` prop and based on that, the <SocialButton /> component will get adjust accordingly... also we will get rid of useless import s across the app